server/api/standardized_contests.py

```diff
@@ -177,43 +178,57 @@ def process_standardized_contests_file(election_id: str):
     set_contest_metadata(election)
 
 
-def validate_standardized_contests_upload(request: Request, election: Election):
-    if election.audit_type not in [AuditType.BALLOT_COMPARISON, AuditType.HYBRID]:
-        raise Conflict("Can't upload CVR file for this audit type.")
-
-    if len(list(election.jurisdictions)) == 0:
-        raise Conflict(
-            "Must upload jurisdictions file before uploading standardized contests file."
-        )
-
-    if "standardized-contests" not in request.files:
-        raise BadRequest("Missing required file parameter 'standardized-contests'")
+@api.route(
+    "/election/<election_id>/standardized-contests/file/upload-url", methods=["GET"]
+)
+@restrict_access([UserType.AUDIT_ADMIN])
+def start_upload_for_standardized_contests_file(election: Election):
+    file_type = request.args.get("fileType")
+    if file_type is None:
+        raise BadRequest("Missing expected query parameter: fileType")
 
-    validate_csv_mimetype(request.files["standardized-contests"])
+    storage_path_prefix = f"audits/{election.id}"
+    filename = timestamp_filename("standardized_contests", "csv")
 
+    return jsonify(get_file_upload_url(storage_path_prefix, filename, file_type))
 
-@api.route("/election/<election_id>/standardized-contests/file", methods=["PUT"])
-@restrict_access([UserType.AUDIT_ADMIN])
-def upload_standardized_contests_file(election: Election):
-    validate_standardized_contests_upload(request, election)
 
-    election.standardized_contests = None
-    file = request.files["standardized-contests"]
-    storage_path = store_file(
-        file.stream,
-        f"audits/{election.id}/" + timestamp_filename("standardized_contests", "csv"),
-    )
+def save_standardized_contests_file(
+    election: Election, storage_path: str, filename: str
+):
     election.standardized_contests_file = File(
         id=str(uuid.uuid4()),
-        name=file.filename,  # type: ignore
+        name=filename,  # type: ignore
         storage_path=storage_path,
         uploaded_at=datetime.now(timezone.utc),
     )
     election.standardized_contests_file.task = create_background_task(
         process_standardized_contests_file, dict(election_id=election.id)
     )
-    db_session.commit()
 
+
+@api.route(
+    "/election/<election_id>/standardized-contests/file/upload-complete",
+    methods=["POST"],
+)
+@restrict_access([UserType.AUDIT_ADMIN])
+def complete_upload_for_standardized_contests_file(election: Election):
+    if election.audit_type not in [AuditType.BALLOT_COMPARISON, AuditType.HYBRID]:
+        raise Conflict("Can't upload CVR file for this audit type.")
```

Comment:
Weird, I think this error message should say "Can't upload standardized contests file for this audit type." Looks like an existing bug. Also doesn't really matter since the UI doesn't allow it